wolfhsm/wh_server.h

```diff
@@ -66,22 +126,53 @@ typedef struct whServerConfig_t {
     whServerDmaConfig* dmaConfig;
 } whServerConfig;
 
+
+/* Context structure to maintain the state of an HSM server */
+struct whServerContext_t {
+    whCommServer comm[1];
+    whNvmContext* nvm;
+    crypto_context* crypto;
+    CacheSlot cache[WOLFHSM_NUM_RAMKEYS];
+    whServerCustomCb customHandlerTable[WH_CUSTOM_CB_NUM_CALLBACKS];
+    whServerDmaContext dma;
+    int connected;
+};
+
+
+/** Public server context functions */
+
 /* Initialize the comms and crypto cache components.
- * Note: NVM and Crypto must be initialized prior to Server Init
+ * Note: NVM and Crypto components must be initialized prior to Server Init
```

Comment:
Still pretty vauge, maybe `hardware drivers`

Reply:
Server has multiple components: comm, nvm, crypto, customcb, and eventually SHE and PKCS11, so these aren't necessarily drivers.  We need to ensure that if these pointers are NULL or not initialized, that the server gracefully handles this case.  Added to backlog.